app/uk/gov/hmrc/servicedependencies/model/MetaArtefactDependency.scala

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.servicedependencies.model
+
+import play.api.libs.functional.syntax.toFunctionalBuilderOps
+import play.api.libs.json.Format.GenericFormat
+import play.api.libs.json._
+import uk.gov.hmrc.servicedependencies.model.RepoType.Other
+import uk.gov.hmrc.servicedependencies.service.DependencyService
+
+case class MetaArtefactDependency(
+                       slugName: String,
+                       slugVersion: Version,
+                       teams: List[String], // Override on write
+                       repoType: RepoType,
+                       group: String,
+                       artefact: String,
+                       artefactVersion: Version,
+                       compileFlag: Boolean,
+                       providedFlag: Boolean,
+                       testFlag: Boolean,
+                       itFlag: Boolean,
+                       buildFlag: Boolean
+                     )
+
+object MetaArtefactDependency {
+
+  implicit val versionFormats: Format[Version] = Version.format
+
+  val mongoFormat: OFormat[MetaArtefactDependency] = {
+    ((__ \ "slugName").format[String]
+      ~ (__ \ "slugVersion").format[Version]
+      ~ (__ \ "repoType").format[RepoType]
+      ~ (__ \ "group").format[String]
+      ~ (__ \ "artefact").format[String]
+      ~ (__ \ "artefactVersion").format[Version]
+      ~ (__ \ "compileFlag").format[Boolean]
+      ~ (__ \ "providedFlag").format[Boolean]
+      ~ (__ \ "testFlag").format[Boolean]
+      ~ (__ \ "itFlag").format[Boolean]
+      ~ (__ \ "buildFlag").format[Boolean]
+      )((sn, sv, rt, g, a, av, cf, pf, tf, itf, bf) =>
+      MetaArtefactDependency(
+        sn, sv, List.empty, rt, g, a, av, cf, pf, tf, itf, bf
+      ),
+      ( mad => (mad.slugName,
+        mad.slugVersion,
+        mad.repoType,
+        mad.group,
+        mad.artefact,
+        mad.artefactVersion,
+        mad.compileFlag,
+        mad.providedFlag,
+        mad.testFlag,
+        mad.itFlag,
+        mad.buildFlag
+      ))
+    )
+  }
+
+
+  val apiWrites: OWrites[MetaArtefactDependency] = {
+
+    implicit val depScope   = DependencyScope.dependencyScopeFormat
+
+    (
+      (__ \ "slugName").write[String] and
```

Comment:
We usually only use infix symbols - not words. Here replacing `and` with `~` on the left is easier to see the shape of the data independently from the values.